easybuild/easyconfigs/t/Trilinos/Trilinos-13.4.1-foss-2022a.eb

```
@@ -0,0 +1,48 @@
+# easyblock = 'CMakeMake'
+
+name = 'Trilinos'
+version = '13.4.1'
+
+homepage = 'https://trilinos.org'
+description = """The Trilinos Project is an effort to develop algorithms and enabling technologies
+ within an object-oriented software framework for the solution of large-scale, complex multi-physics
+ engineering and scientific problems. A unique design feature of Trilinos is its focus on packages."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'usempi': True, 'pic': True, 'strict': True}
+
+source_urls = ['https://github.com/trilinos/Trilinos/archive/refs/tags/']
+sources = ['trilinos-release-13-4-1.tar.gz']
```

@branfosj We should probably avoid the hardcoded version here?

```suggestion
sources = ['trilinos-release-%star.gz' % '-'.join(version.split('.'))]